src/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -12,10 +12,10 @@ public abstract partial class AsymmetricAlgorithm : System.IDisposable
         protected int KeySizeValue;
         protected System.Security.Cryptography.KeySizes[] LegalKeySizesValue;
```

Comment:
Shouldn't this be nullable?

Reply:
This was always set in derived classes and anywhere always referenced without null check.  Seems intent is non null and derived classes should set this value with non null instance, I thought if make it nullable devs might think they can set it null in derived class